### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Maahi10001**
## Description

Adding emulator work flow and small document changes.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Sarfraz-droid** [comment]
Should we add the Flowchart from https://github.com/scorelab/Codelabz/pull/401 to the documentation?

**Review by Sarfraz-droid — COMMENTED** [review]
Should we add the Flowchart from https://github.com/scorelab/Codelabz/pull/401 to the documentation?

**Comment by Maahi10001** [comment]
> Should we add the Flowchart from #401 to the documentation?
yeah sure ,that'll be good to understand the flow also

**Comment by Sarfraz-droid** [comment]
> > Should we add the Flowchart from #401 to the documentation?
> > yeah sure ,that'll be good to understand the flow also

👍

File: README.md [hunk 1]
```diff
@@ -110,8 +110,18 @@ It will redirect to 6006 port.Find detailed information [here](https://storybook
 To run cypress tests:
 `npx cypress run`
 It will open a promt displaying all the tests.You can find detailed information [here](https://docs.cypress.io/guides/guides/command-line#How-to-run-commands)
-
-
+# Firebase Base Emulator Flow
```
Comment (Sarfraz-droid) [inline README.md]: Should it be "Firebase Emulator flow" instead of "Firebase Base"?

**Review by Sarfraz-droid — APPROVED** [review]

**Review by sakkshm26 — APPROVED** [review]

**Review by shahaab — APPROVED** [review]

**Pull request merged**